.gitignore

```diff
@@ -2,9 +2,22 @@
 __pycache__
 /TEMP
 *.pyd
-hubert_base.pt
-/logs
 .venv
 /opt
 tools/aria2c/
 tools/flag.txt
+
+# Imported from huggingface.co/lj1995/VoiceConversionWebUI
+/pretrained
+/pretrained_v2
+/uvr5_weights
+hubert_base.pt
+rmvpe.onnx
+rmvpe.pt
+
+# Generated by RVC
+/logs
+/weights
```

Comment:
These directories have been moved and each folder has a .gitigone.

Reply:
Current users might have a very bad surprise when they `git pull` and `git checkout -- .` to get the latest changes; their entire models will be wiped with no way to recover them!

Even though they have been moved in the current version, it's important to keep these entries here to prevent disasters. Plus, one would expect to look at these files and automatically move them to `assets` on the next launch to ease migration from versions.

Reply:
I agree with you.